node/node.go

```diff
@@ -120,17 +121,25 @@ func (nn *nexNode) Validate() error {
 	return errs
 }
 
-func (nn *nexNode) Start() {
-	nn.initializeSupervisionTree()
+func (nn *nexNode) Start() error {
+	return nn.initializeSupervisionTree()
 }
 
-func (nn *nexNode) initializeSupervisionTree() {
+func (nn *nexNode) initializeSupervisionTree() error {
 	var options gen.NodeOptions
 
+	nodeName := "nex@localhost" // FIXME-- make configurable
```

Comment:
I actually think this shouldn't be configurable. This is only used for ergo internals that make use of a clustering feature we don't use.